airlock/views/request.py

```diff
@@ -372,6 +372,8 @@ def request_view(request, request_id: str, path: str = ""):
         "include_code": code_url is not None,
         "include_download": not is_author,
         "upload_in_progress": release_request.upload_in_progress(),
+        "is_htmx_request": request.htmx,
```

Comment:
We have access to `request` in the template, is there a benefit to making `is_htmx_request` a separate variable? Does the template linter complain?

Reply:
Good point. I'll change.